server/api/user/user.controller.js

```diff
@@ -34,7 +35,7 @@ exports.index = function(req, res) {
 exports.search = function(req, res){
     if (!req.query.query) return res.send(400, "No query supplied");
     var query = new RegExp(["^", req.query.query, "$"].join(""), "i")
-    User.findOne({name: query}, function(err, user){
+    User.findOne({name: query},{active:{$nin:false}},function(err, user){
```

Comment:
what is this doing here? You shouldn't be updating search code for deleting a user should you?